examples/juniper/cptx-host/cptx-host.pb.txt

```diff
@@ -5,12 +5,13 @@ nodes: {
     model: "cptx"
     os: "evo"
     config: {
+      image: "cptx:test"
```

Comment:
why add this?

Reply:
Just to keep this in sync with the other cptx example cptx-ixia/cptx-ixia.pb.txt

Reply:
Can you remove both this and the image field in cptx-ixia/cptx-ixia.pb.txt?

I would prefer to keep the examples using the node implementation defaults unless there is a specific version required to run the example (which in this case there is not)

Reply:
makes sense. Done.